.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

Comment:
Please, drop this comment.
We use dependabot in many projects already, so it is not something new to us.